### PR TITLE
Bump go-build-tools to v0.1.0

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

File: tools/go.mod [hunk 1]
```diff
@@ -154,12 +153,12 @@ require (
 	github.com/spf13/cobra v1.5.0 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/spf13/viper v1.12.0 // indirect
+	github.com/spf13/viper v1.13.0 // indirect
```
Comment (MadVikingGod) [inline tools/go.mod:156]: Looks like this is the culprit.  We can add a replace back to v1.12.0 until golangci-lint fixes this behavior.

**Comment by bogdandrutu** [comment]
@MadVikingGod do you want me to add the replace?

**Comment by MrAlias** [comment]
Looks like viper does not have a plan to fix the issue: https://github.com/spf13/viper/issues/1431

**Comment by MadVikingGod** [comment]
The community agreed that disabling the linter, for the time being, is the better option.  #2878 is the fix specifically for that.  This can merge after.

**Comment by pellared** [comment]
This PR is outdated.

**Pull request closed without merging**